course_flow/models.py

```diff
@@ -211,42 +216,49 @@ class Meta:
         verbose_name = "Node-Strategy Link"
         verbose_name_plural = "Node-Strategy Links"
 
-
-class Activity(models.Model):
+        
+class Workflow(models.Model):
+    objects = InheritanceManager()
+    
     title = models.CharField(max_length=30)
     description = models.TextField(max_length=400)
-    author = models.ForeignKey(
-        User,
-        related_name="authored_activities",
-        on_delete=models.SET_NULL,
-        null=True,
-    )
     created_on = models.DateTimeField(auto_now_add=True)
     last_modified = models.DateTimeField(auto_now=True)
 
     static = models.BooleanField(default=False)
 
-    students = models.ManyToManyField(
-        User, related_name="assigned_activities", blank=True
-    )
-
     parent_activity = models.ForeignKey(
```

Comment:
change this to `parent_workflow`

Reply:
Right, good call, I meant to do this after all the other migrations but forgot. Can't even remember why I didn't change it right away.